docs/UserGuide.md

```diff
@@ -404,11 +404,91 @@ Examples:
 * `importcsv n/2`
   * Reads from CSV treating the 2nd row as names, 3rd row as phone numbers, 4th row as email address, 5th row as tagged.
 
-
 ### Archiving data files `[coming in v2.0]`
 
 _Details coming soon ..._
 
+###Extension on Find contact: `find` `t/`
+
+Locates persons by different search types
+
+Format: `t/SEARCH_TYPE k/KEYWORD [MORE_KEYWORDS]`
+
+* The search is case-insensitive. e.g. hans will match Hans
+* The order of the keywords does not matter. e.g. Hans Bo will match Bo Hans
+* `SEARCH_TYPE` must match one of the following:
+  * Name
+  * Address
+  * Phone
+  * Email
+* Persons matching at least one keyword will be returned
+
+Examples:
+* `find t/Address k/Garden`
+    * Lists all contact with address contains keyword “Garden”
+* `find t/Phone k/87438807`
+    * Lists all contact with phone matches keyword “87438807”
+
+### Batch and Range Commands
+
+### Fetch contacts by Condition Clause: `where/`
+
+If the given condition is satisfied, it returns a group of contacts
+
+Format: `batch COMMAND where/CONDITION`
+
+* The commands include list, edit, delete
+* The condition is expressed using logical operators
+  * `>`  (compare integer)
+  * `<`  (compare integer)
+  * `=` (compare integer or string value)
+  * `LIKE`
+    * `%`  wildcard start or ends with pattern
+  * `NOT` Combines with other logical operators to perform negation
+
+Examples:
+* `batch delete where/ n/NAME LIKE %a`
```

Comment:
This will not work under the tokenizer system, `where/` will be empty.
```suggestion
* `batch delete where/name LIKE a%` Deletes all persons whose name start with a.
```